benchmarks/hia/countdown/Main.hs

```diff
@@ -13,9 +13,9 @@ import HIA.DesugarHandlers
 [operation|Put s :: s -> ()|]
 
 [handler|
-    RunState s a :: s -> (a, s) 
+    RunState s a :: s -> a
```

Comment:
Nit: this is typically called `EvalState` in the functional programming literature. The three variations:
* `RunState s a -> s -> (a, s)`
* `EvalState s a -> s -> a`
* `ExecState s a -> s -> s`

Reply:
Renamed it now, I had updated the types and results but not the function names.